IdokladSdk/Requests/DocumentPayment/Sales/Sort/SalesPaymentSort.cs

```diff
@@ -0,0 +1,19 @@
+﻿using IdokladSdk.Models.Payment.DocumentPayments.Sales.List;
+using IdokladSdk.Requests.Core.Modifiers.Sort.BasicSorts;
+using IdokladSdk.Requests.Core.Modifiers.Sort.Common;
+
+namespace IdokladSdk.Requests.DocumentPayment.Sales.Sort
+{
+    /// <summary>
+    /// Sortable properties of <see cref="SalesDocumentPaymentListGetModel"/>.
+    /// </summary>
+    public class SalesPaymentSort : IdSort
+    {
+        /// <inheritdoc cref="SalesDocumentPaymentListGetModel.DateOfPayment" />
+        public SortItem DateOfPayment { get; set; } =
+            new SortItem(nameof(SalesDocumentPaymentListGetModel.DateOfPayment));
+
+        /// <inheritdoc cref="SalesDocumentPaymentListGetModel.PaidDocument" />
+        public SortItem DocumentId { get; set; } = new SortItem("DocumentId");
```

Comment:
Včera jsme se bavili, že na to v API udělám interface, který bude ten GET model implementovat a pak tu v sortu použiju nameof. Udělal bych tady to samé.

Reply:
Probráno osobně. Vytvořeny aspoň testy.